src/app/Enums/UpdateType.php

```diff
@@ -9,4 +9,11 @@ class UpdateType extends Enum
     const Single = 'single';
```

Comment:
let;s replace the options with

- only this event
- this and future events
- all